modtools/create-item.lua

```diff
@@ -239,38 +240,51 @@ local function createCorpsePiece(creator, bodypart, partlayer, creatureID, caste
 end
 
 local function createItem(mat, itemType, quality, creator, description, amount)
-    local item = df.item.find(dfhack.items.createItem(itemType[1], itemType[2], mat[1], mat[2], creator))
-    local item2 = nil
-    assert(item, ('failed to create item: item_type: %s, item_subtype: %s, mat_type: %s, mat_index: %s, unit: %s'):format(itemType[1], itemType[2], mat[1], mat[2], creator and creator.id or 'nil'))
+    -- The "reaction-gloves" tweak can cause this to create multiple gloves
+    local items = dfhack.items.createItem(creator, itemType[1], itemType[2], mat[1], mat[2])
+    assert(#items, ('failed to create item: item_type: %s, item_subtype: %s, mat_type: %s, mat_index: %s, unit: %s'):format(itemType[1], itemType[2], mat[1], mat[2], creator and creator.id or 'nil'))
```

Comment:
`#items` is always true. the only values interpreted as false are `false` and `nil`. this should be `#items > 0`